README.md

```diff
@@ -10,81 +10,62 @@ This is composed by:
 
 > The example is intended to be deployed and run in an Openshift or [CodeReady containers](https://developers.redhat.com/products/codeready-containers/overview) with [Kafka operator](https://strimzi.io/quickstarts/) installed. (Although it is possible to run on a common kafka installation.)
 
+## Setup Kafka Operator
+
+Installing the Kafka Operator using the Web Console of Openshift is pretty straightforward.
+
+First, go to the *Operator Hub* and search for *Kafak*.  A few choices might show up,  but we want the one that says "Strimzi" which is the community Kafka version.
+
+Since we just want the defaults, we can just click *install* on the next 2 screens.  Then wait until the operator is ready.
+
+Next, create a new project.  I used nodejs-examples, but it can be anything.  This can either be done on the command line with the `oc` client, or in the web console.  I tend to do this step in the console.
+
+```
+$ oc new-project nodejs-examples
+```
+
+The last step of the setup, is to create a new Kafka instance.  This can be done from the web console also.  Navigate to the installed operators and click on the kafka one that was just installed.
+
+Before clicking on the *Create Instance* link under the Kafka tab, make sure your "current project namespace" is the one you just created.  Once that is set, click the *Create Instance* link and just use the defaults, which should name the new cluster `my-cluster`
+
+## Deploy the Applications
+
 Both producer and consumer are using [node-rdkafka](https://github.com/Blizzard/node-rdkafka).
 
 ### producer-backend
 
-The producer-backend is using the [`Producer` constructor](https://github.com/Blizzard/node-rdkafka#sending-messages) to 
+The producer-backend is using the [`Producer` constructor](https://github.com/Blizzard/node-rdkafka#sending-messages) to
 send messages to Kafka, which its content is a random country name.
 
-A namespace `nodejs-kafka` was created in Openshift to deploy the tree services. So the producer's `bootstrap.servers` are 
-configured like this:
+If you used the defaults for the cluster naming, then nothing needs to change, but if you named the cluster something else, then you will need to change the server name in the code:
 
 ```
 new Kaka.Producer({
-  'bootstrap.servers': 'nodejs-kafka-cluster-kafka-bootstrap:9092'
+  'bootstrap.servers': 'my-cluster-kafka-bootstrap:9092'
 })
 ```
```

Comment:
This needs to be updated to reflect the changes you made to add `process.env.KAFKA_BOOTSTRAP_SERVER`.  I think we can also say in the text above something like "you may have to update the server name. Some environments like openshift will automatically inject the server name, if that's not the case for you environment you'll need to update as follows"